config/params.ini

```diff
@@ -75,6 +75,8 @@ belt_rejuv_columns=2
 belt_hp_columns=1
 belt_mp_columns=1
 pre_buff_every_run=0
+always_repair=0
```

Comment:
Always repair lines probably shouldn't be included?

Reply:
This is part of the "This branch has conflicts that must be resolved" that i was trying to get rid of.